draft-ietf-tls-tls13.md

```diff
@@ -4906,7 +4906,7 @@ Peer Authentication.
 identity. If the client is authenticated, the server's view of the
 peer identity should match the client's identity.
 
-Uniqueness of the session key:
+Uniqueness of the session keys:
```

Comment:
The need for uniqueness applies not to the set, but to every individual key in that set.